arrow-fx-coroutines/src/test/kotlin/arrow/fx/coroutines/ParTraverseTest.kt

```diff
@@ -1,9 +1,11 @@
 package arrow.fx.coroutines
 
 import arrow.core.Either
+import io.kotest.assertions.throwables.shouldThrowExactly
 import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 import io.kotest.property.Arb
+import io.kotest.property.arbitrary.constant
```

Comment:
```suggestion
```

Unused import.

Reply:
Sorry. I just am working with a half setup project. Files are taking long to get analyzed and show different colors on different things and infer types hints, imports.

Reply:
Oh weird. I don't seem to have that issue. I have inferred type hints disabled tho, and some other hints as well and I gave IDEA CE an excessive amount of RAM.